tests/stubtest_allowlists/py3_common.txt

```diff
@@ -392,6 +393,13 @@ email.contentmanager.set_bytes_content
 email.contentmanager.set_message_content
 email.contentmanager.set_text_content
 email.iterators.walk
+pipes.Template.makepipeline
```

Comment:
Why was this moved?

Reply:
Undocumented and there's no docstring, seems like it belongs in the "undocumented implementation details" section?

Reply:
Happy to add it to the stub if you think it's useful

Reply:
Seems to me like it's just a helper function used by `open_r`, `open_w` and `copy`